.pre-commit-config.yaml

```diff
@@ -0,0 +1,28 @@
+repos:
+- hooks:
+  - {id: seed-isort-config, language_version: python3.7}
+  repo: https://github.com/asottile/seed-isort-config
+  rev: v1.9.2
+- hooks:
+  - {id: isort, language_version: python3.7}
+  repo: https://github.com/pre-commit/mirrors-isort
+  rev: v4.3.21
+- hooks:
+  - {id: black, language_version: python3.7}
+  repo: https://github.com/ambv/black
+  rev: 19.3b0
+- hooks:
+  - {id: trailing-whitespace, language_version: python3.7}
+  - {id: end-of-file-fixer, language_version: python3.7}
+  - {id: debug-statements, language_version: python3.7}
+  - additional_dependencies: [flake8-bugbear == 18.8.0, flake8-per-file-ignores]
+    id: flake8
+    language_version: python3.7
+  repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.2.3
+- hooks:
+  - id: pyupgrade
+    language_version: python3.7
+    # args: [--py3-plus, --py36-plus]
```

Comment:
This would use advanced language features such as `f"{}"` string substitution... Not for now.